packages/venia-concept/_buildpack/create.js

```diff
@@ -91,6 +91,7 @@ function createProjectFromVenia({ fs, tasks, options }) {
                     pkg.scripts[name] = toPackageScript(scriptsToInsert[name]);
                 });
 
+                // eslint-disable-next-line no-extra-boolean-cast
```

Comment:
Rather than require a regression for code changes I was just adding disables where new eslint warnings were occurring in the app at present.

If you want me to do this you'll need to verify it doesn't break anything.